src/Aquifer.API/Endpoints/Projects/Start/Endpoint.cs

```diff
@@ -1,8 +1,10 @@
 using Aquifer.API.Common;
 using Aquifer.API.Services;
-using Aquifer.Common.Services;
+using Aquifer.Common.Jobs.Messages;
+using Aquifer.Common.Jobs.Publishers;
 using Aquifer.Data;
 using Aquifer.Data.Entities;
+using Aquifer.Data.Services;
```

Comment:
I moved the `ResourceHistoryService` into the `Aquifer.Data` project so that I could access it from `Aquifer.Jobs`.